sdk/servicebus/service-bus/test/receiveAndDeleteMode.spec.ts

```diff
@@ -243,17 +243,17 @@ describe("receive and delete", () => {
         : TestMessage.getSample();
       // we have to force this cast - the type system doesn't allow this if you've chosen receiveAndDelete
       // as your lock mode.
-      const msg = (await sendReceiveMsg(testMessages)) as ServiceBusReceivedMessageWithLock;
+      const msg = (await sendReceiveMsg(testMessages)) as ServiceBusReceivedMessage;
```

Comment:
This type cast is not required anymore I guess.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/11962/commits/11c467b6ee3d2d44d0c781d14e9b172d1a56ac0d